book/quick-brick/Setup.md

```diff
@@ -5,57 +5,151 @@
+
+### Add a plugin to a QuickBrick workspace
+
+When creating a plugin, follow these steps:
+
+- In your forked [QuickBrick](https://github.com/applicaster/QuickBrick) repository, create a folder for your plugin project. The folder should be located inside the `plugins` directory. 
+- Run `npm init` to initialize your package.json file for your plugin.
+- The `package.json` file should contain the following base values:
+
+    ```json
+    {
+        "name": "@your_npm_account/plugin_name",
+        "version": "1.0.0",
+        "description": "plugin_description",
+        "main": "index.js",
+        "scripts": {
+            "test": "jest"
+        },
+        "author": "",
+        "license": "ISC",
+        "peerDependencies": {
+            "react": "16.8.3",
+            "react-native": "0.59.10",
+        },
+        "devDependencies": {
+            "react": "16.8.3",
+            "react-native": "0.59.10",
+        },
+        "dependencies": {
+            "react": "16.8.3",
+            "react-native": "0.59.10"
+        },
+        "jest": {
+            "preset": "react-native"
+        },
+        "applicaster": {
+            "supportedPlatforms": [
+                "ios",
+                "android"
+            ]
+        }
+    }
+    ```
+    Notes:
+    - A `main` property in your `package.json` points to the file where your plugin's entry point will be. This is the file which will be reached when importing your plugin through NPM. You can change it at any time.
+    - The `license` type Varies between plugins.
+    - Your plugin can support multiple platforms, please specify it in the `supportedPlatforms` JSON array.
+- Add a `.npmignore` file for all the files you don't want to push to npm (like the QuickBrick bootstrap files for instance). We suggests to add [iOS](https://github.com/github/gitignore/blob/master/Global/Xcode.gitignore), [Android](https://github.com/github/gitignore/blob/master/Android.gitignore) and [react native](https://github.com/facebook/react-native/blob/master/.gitignore) git ignores.
+- Edit `index.js` to import your plugin code, and either push it to the plugin's array, or in the `components` property of the `createZappApp` function call if your plugin is a Ui Builder `ui_component`.
+- Manually update the file `config/rivers.json` to insert data which will result in your plugin showing up in the app. If you're not sure how to achieve this, ask #support or a member of the Zapp Team.
+
+### iOS/tvOS Quick-Brick
+
+The following chapter will explain how to create and prepare a project for iOS/tvOS platforms with quick-brick. Please use it when you need to develop or test your iOS/tvOS apps.
+
+### Prerequisites for iOS/tvOS with Quick-Brick
+
```

Comment:
We now have an init command on the zapplicaster-cli which clones all that is needed and saves a config file. It’s in the read me I sent you. Once this is done, the build mobile flag can be used in the prepare command to bootstrap and build the native app as needed